
=CRASH REPORT==== 30-Mar-2010::15:01:45 ===
  crasher:
    pid: <0.6651.1778>
    registered_name: []
    exception exit: {system_limit,
                        [{erlang,spawn_opt,
                             [proc_lib,init_p,
                              [<0.6651.1778>,
                               [<0.2201.1778>],
                               gen,init_it,
                               [gen_server,<0.6651.1778>,<0.6651.1778>,
                                rabbit_limiter,
                                [<0.6651.1778>],
                                []]],
                              [link]]},
                         {proc_lib,start_link,5},
                         {rabbit_limiter,start_link,1},
                         {rabbit_channel,handle_method,3},
                         {rabbit_channel,handle_cast,2},
                         {gen_server2,handle_msg,7},
                         {proc_lib,init_p,5}]}
      in function  gen_server2:terminate/6
    initial call: gen:init_it(gen_server2,<0.2201.1778>,<0.2201.1778>,
                              rabbit_channel,
                              [7348,<0.19214.1694>,<0.6708.1777>,
                               <<"stormcloud">>,<<"/">>],
                              [])
    ancestors: [<0.2201.1778>]
    messages: []
    links: [<0.2201.1778>,<0.6708.1777>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 987
    stack_size: 23
    reductions: 355
  neighbours:
    neighbour: [{pid,<0.6708.1777>},
                  {registered_name,[]},
                  {initial_call,{rabbit_writer,mainloop,1}},
                  {current_function,{rabbit_writer,mainloop,1}},
                  {ancestors,[]},
                  {messages,[shutdown]},
                  {links,[<0.6651.1778>]},
                  {dictionary,[]},
                  {trap_exit,false},
                  {status,runnable},
                  {heap_size,233},
                  {stack_size,2},
                  {reductions,17}]

=CRASH REPORT==== 31-Mar-2010::01:57:08 ===
  crasher:
    pid: <0.11.0>
    registered_name: rex
    exception exit: {system_limit,
                        [{erlang,spawn_opt,
                             [{erlang,apply,
                                  [#Fun<rpc.0.69165610>,[]],
                                  [monitor]}]},
                         {erlang,spawn_monitor,1},
                         {rpc,handle_call,3},
                         {gen_server,handle_msg,5},
                         {proc_lib,init_p,5}]}
      in function  gen_server:terminate/6
    initial call: gen:init_it(gen_server,<0.10.0>,<0.10.0>,
                              {local,rex},
                              rpc,[],[])
    ancestors: [kernel_sup,<0.9.0>]
    messages: []
    links: [<0.10.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 121393
    stack_size: 23
    reductions: 15640812
  neighbours:

=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,kernel_sup}
     Context:    child_terminated
     Reason:     {system_limit,
                     [{erlang,spawn_opt,
                          [{erlang,apply,
                               [#Fun<rpc.0.69165610>,[]],
                               [monitor]}]},
                      {erlang,spawn_monitor,1},
                      {rpc,handle_call,3},
                      {gen_server,handle_msg,5},
                      {proc_lib,init_p,5}]}
     Offender:   [{pid,<0.11.0>},
                  {name,rex},
                  {mfa,{rpc,start_link,[]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,kernel_sup}
     Context:    shutdown
     Reason:     reached_max_restart_intensity
     Offender:   [{pid,<0.11.0>},
                  {name,rex},
                  {mfa,{rpc,start_link,[]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    child_terminated
     Reason:     shutdown
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=CRASH REPORT==== 31-Mar-2010::01:57:08 ===
  crasher:
    pid: <0.11.1833>
    registered_name: memsup
    exception exit: {shutdown,
                        {gen_server,call,
                            [kernel_safe_sup,
                             {start_child,
                                 {timer_server,
                                     {timer,start_link,[]},
                                     permanent,1000,worker,
                                     [timer]}},
                             infinity]}}
      in function  gen_server:init_it/6
    initial call: gen:init_it(gen_server,<0.52.0>,<0.52.0>,
                              {local,memsup},
                              rabbit_memsup,
                              [rabbit_memsup_linux],
                              [])
    ancestors: [os_mon_sup,<0.51.0>]
    messages: []
    links: [<0.52.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 377
    stack_size: 23
    reductions: 135
  neighbours:

=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    start_error
     Reason:     {shutdown,
                     {gen_server,call,
                         [kernel_safe_sup,
                          {start_child,
                              {timer_server,
                                  {timer,start_link,[]},
                                  permanent,1000,worker,
                                  [timer]}},
                          infinity]}}
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=CRASH REPORT==== 31-Mar-2010::01:57:08 ===
  crasher:
    pid: <0.98.1833>
    registered_name: memsup
    exception exit: {noproc,
                        {gen_server,call,
                            [kernel_safe_sup,
                             {start_child,
                                 {timer_server,
                                     {timer,start_link,[]},
                                     permanent,1000,worker,
                                     [timer]}},
                             infinity]}}
      in function  gen_server:init_it/6
    initial call: gen:init_it(gen_server,<0.52.0>,<0.52.0>,
                              {local,memsup},
                              rabbit_memsup,
                              [rabbit_memsup_linux],
                              [])
    ancestors: [os_mon_sup,<0.51.0>]
    messages: []
    links: [<0.52.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 377
    stack_size: 23
    reductions: 127
  neighbours:

=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    start_error
     Reason:     {noproc,
                     {gen_server,call,
                         [kernel_safe_sup,
                          {start_child,
                              {timer_server,
                                  {timer,start_link,[]},
                                  permanent,1000,worker,
                                  [timer]}},
                          infinity]}}
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=CRASH REPORT==== 31-Mar-2010::01:57:08 ===
  crasher:
    pid: <0.146.1833>
    registered_name: memsup
    exception exit: {noproc,
                        {gen_server,call,
                            [kernel_safe_sup,
                             {start_child,
                                 {timer_server,
                                     {timer,start_link,[]},
                                     permanent,1000,worker,
                                     [timer]}},
                             infinity]}}
      in function  gen_server:init_it/6
    initial call: gen:init_it(gen_server,<0.52.0>,<0.52.0>,
                              {local,memsup},
                              rabbit_memsup,
                              [rabbit_memsup_linux],
                              [])
    ancestors: [os_mon_sup,<0.51.0>]
    messages: []
    links: [<0.52.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 377
    stack_size: 23
    reductions: 131
  neighbours:

=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    start_error
     Reason:     {noproc,
                     {gen_server,call,
                         [kernel_safe_sup,
                          {start_child,
                              {timer_server,
                                  {timer,start_link,[]},
                                  permanent,1000,worker,
                                  [timer]}},
                          infinity]}}
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=CRASH REPORT==== 31-Mar-2010::01:57:08 ===
  crasher:
    pid: <0.432.1833>
    registered_name: memsup
    exception exit: {noproc,
                        {gen_server,call,
                            [kernel_safe_sup,
                             {start_child,
                                 {timer_server,
                                     {timer,start_link,[]},
                                     permanent,1000,worker,
                                     [timer]}},
                             infinity]}}
      in function  gen_server:init_it/6
    initial call: gen:init_it(gen_server,<0.52.0>,<0.52.0>,
                              {local,memsup},
                              rabbit_memsup,
                              [rabbit_memsup_linux],
                              [])
    ancestors: [os_mon_sup,<0.51.0>]
    messages: []
    links: [<0.52.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 377
    stack_size: 23
    reductions: 135
  neighbours:

=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    start_error
     Reason:     {noproc,
                     {gen_server,call,
                         [kernel_safe_sup,
                          {start_child,
                              {timer_server,
                                  {timer,start_link,[]},
                                  permanent,1000,worker,
                                  [timer]}},
                          infinity]}}
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]


=SUPERVISOR REPORT==== 31-Mar-2010::01:57:08 ===
     Supervisor: {local,os_mon_sup}
     Context:    shutdown
     Reason:     reached_max_restart_intensity
     Offender:   [{pid,<0.146.0>},
                  {name,memsup},
                  {mfa,{rabbit_memsup,start_link,[rabbit_memsup_linux]}},
                  {restart_type,permanent},
                  {shutdown,2000},
                  {child_type,worker}]

